e2e_test/ch-benchmark/create_sources.slt.part

```diff
@@ -0,0 +1,181 @@
+statement ok
+CREATE MATERIALIZED SOURCE tpcc_customer (
```

Comment:
> TPC-C tables are explicitly prefixed tpcc_. This is in order to avoid confusion since both TPC-H and TPC-C have a customers and order(s) table.


The renaming doesn't seem to be necessary because we create & drop tables for each test cases. Correct me if wrong

Reply:
It is because ch-benchmark also utilizes some TPC-H tables. I include those in `e2e_test/ch-benchmark/ch_benchmark.slt` like this `include ../tpch/create_tables.slt.part`. `../tpch/create_tables.slt.part` creates all TPC-H tables so also those with duplicate names (even though they're not actually used in the ch-benchmark. 

So in short, in the current situation both `customer` and `orders` tables are created (TPC-H) as well as `tpcc_customer` and `tpcc_orders` of which only the latter two are actually used.